lib/sdr_client/update.rb

```diff
@@ -27,93 +27,114 @@ def run
 
     attr_reader :druid, :logger, :options, :url
 
-    def updated_cocina_item
-      @updated_cocina_item ||=
-        original_cocina_item.then { |cocina_item| update_apo(cocina_item) }
-                            .then { |cocina_item| update_collection(cocina_item) }
-                            .then { |cocina_item| update_copyright(cocina_item) }
-                            .then { |cocina_item| update_use_and_reproduction(cocina_item) }
-                            .then { |cocina_item| update_license(cocina_item) }
-                            .then { |cocina_item| update_access(cocina_item) }
+    def updated_cocina_object
+      @updated_cocina_object ||=
+        original_cocina_object.then { |cocina_object| update_apo(cocina_object) }
+                              .then { |cocina_object| update_collection(cocina_object) }
+                              .then { |cocina_object| update_copyright(cocina_object) }
+                              .then { |cocina_object| update_use_and_reproduction(cocina_object) }
+                              .then { |cocina_object| update_license(cocina_object) }
+                              .then { |cocina_object| update_access(cocina_object) }
+                              .then { |cocina_object| update_cocina(cocina_object) }
```

Comment:
not sure i'm interpreting this correctly, but:  it seems like this chaining means that the cocina in the file will override whatever else may be specified explicitly in other options?  if so, it seems like it might be nice to instead do it the other way, with the `update_cocina` call coming first in the chain, so that explicit CLI arg values get used when provided.

didn't see this behavior specified one way or the other in the ticket, just thinking that my default assumption as a user would be that CLI args would take precedence (or result in some sort of error or warning if they got ignored).

not something i'd block the PR over, as the new functionality seems like an improvement, and i can see arguments for either approach.

Reply:
(feel free to merge if this observation is off base, if this is how the precedence should be, or if you like the idea for the change but would rather do it in a follow-on PR)

Reply:
@jmartin-sul Ah... This is something I considered very briefly when first starting this work... and then completely forgot. Yes, I like what you suggest: doing the cocina update up front in the chain. I'll make that change.. Would you mind requesting changes on this PR to prevent it being accidentally merged? I'll see to this tomorrow. Thanks!

Reply:
done, thanks!